code/controllers/subsystem/persistence.dm

```diff
@@ -358,4 +362,38 @@ SUBSYSTEM_DEF(persistence)
 		if(!original_human || original_human.stat == DEAD || !original_human.all_scars || original_human != ending_human)
 			original_human.save_persistent_scars(TRUE)
 		else
-			original_human.save_persistent_scars()
\ No newline at end of file
+			original_human.save_persistent_scars()
+
+/datum/controller/subsystem/persistence/proc/SaveMuck()
```

Comment:
yeah, they're meant to clean roundstart blood

Reply:
> yeah, they're meant to clean roundstart blood

I meant like map placed blood, it will "save" and then "load" twice when round restarts because it's part of the map and then it loads it again from LoadMuck.

Reply:
ah crap

Reply:
fixed